test/bin/manage_hypervisor_config.sh

```diff
@@ -44,12 +44,25 @@ firewall_settings() {
 
 action_create() {
     # Storage pool
-    if ! sudo virsh pool-info "${VM_STORAGE_POOL}" &>/dev/null; then
-        sudo virsh pool-define-as "${VM_STORAGE_POOL}" dir --target "${VM_DISK_DIR}"
-        sudo virsh pool-build "${VM_STORAGE_POOL}"
-        sudo virsh pool-start "${VM_STORAGE_POOL}"
-        sudo virsh pool-autostart "${VM_STORAGE_POOL}"
-    fi
+    for scenario in "${SCENARIO_SOURCES}"/*.sh; do
```

Comment:
Instead of making this script know which scenarios are being run, could we just have scenario.sh create the storage pool before it creates each VM?

Reply:
I wanted to keep pool management in a single location, but I can change it according to your suggestion if you think it's better.

Reply:
What you have works great in CI, but for a local developer running 1 scenario it's going to end up creating a lot more pools than needed. It also means as the developer adds a new scenario, they have to re-run this script, they can't just run scenario.sh.